drivers/reset/reset_arm_scmi.c

```diff
@@ -190,3 +214,154 @@ static const struct scmi_reset_config scmi_reset_config = {
 
 DEVICE_DT_INST_DEFINE(0, scmi_reset_init, NULL, &scmi_reset_data, &scmi_reset_config, PRE_KERNEL_2,
 		      CONFIG_KERNEL_INIT_PRIORITY_DEFAULT, &scmi_reset_driver_api);
+
+#if defined(CONFIG_ARM_SCMI_RESET_SHELL)
```

Comment:
Why this is part of the driver itself and not part of the `shell.c`?

Reply:
It can't  be moved there as it uses private reset scmi functions.
This scmi reset and shell.c placed in different driver's folders.